pep/package.json

```diff
@@ -82,6 +83,9 @@
         "ember-cli-typescript-blueprints": "^3.0.0",
         "ember-cli-uglify": "^3.0.0",
         "ember-composable-helpers": "^4.2.1",
+        "ember-concurrency": "^1.2.1",
+        "ember-concurrency-decorators": "^2.0.1",
```

Comment:
do we still need ember-concurrency-decorators now that you have ember-concurrency-ts?

Reply:
Due to not being able to get rid of ember-concurrency-decorators as its a peer dep of ember-concurrency-ts, I didn't change anything to use ember-concurrency-ts. But Im leaving both in so we can play around with things